python/paddle/fluid/contrib/mixed_precision/fp16_utils.py

```diff
@@ -72,6 +72,9 @@ def _insert_cast_op(block, op, idx, src_dtype, dest_dtype):
         if src_dtype == core.VarDesc.VarType.FP32 and op.type == 'batch_norm':
             if in_name != 'X':
                 continue
+        if src_dtype == core.VarDesc.VarType.FP32 and op.type == 'fused_bn_add_activation':
+            if in_name != 'X' or in_name != 'Z':
+                continue
```

Comment:
I think the condition test here maybe wrong. Maybe the logic below is more understandable.

```python
if in_name not in {'X', 'Z'}:
    continue
```

Reply:
done.